src/test/resources/data/StudentProfilePageUiTest.json

```diff
@@ -37,13 +37,43 @@
       "isInstructor": false,
       "email": "SHomeUiT.benny.c@gmail.tmt",
       "institute": "TEAMMATES Test Institute 1"
+    },
+    "instructor1OfTestingSanitizationCourse": {
```

Comment:
For future reference, you can discuss the best way to handle this. This was the design decision dated as far back as 2013.